express-crud/server.ts

```diff
@@ -0,0 +1,128 @@
+import express from 'express';
+import pg from 'pg';
+import { ClientError, errorMiddleware } from './lib/index.js';
+
+const db = new pg.Pool({
+  connectionString: 'postgres://dev:dev@localhost/studentGradeTable',
+  ssl: {
+    // Allow non-SSL traffic to localhost
+    rejectUnauthorized: false,
+  },
+});
+
+const app = express();
+
+app.use(express.json());
+
+app.get('/api/grades', async (req, res, next) => {
+  try {
+    const sql = `
+select *
+from "grades"
+`;
+    const result = await db.query(sql);
+    const gradesRow = result.rows;
+    res.status(200).json(gradesRow);
+  } catch (err) {
+    next(err);
+  }
+});
+
+app.get('/api/grades/:gradeId', async (req, res, next) => {
+  try {
+    const { gradeId } = req.params;
+    if (gradeId === undefined)
+      throw new ClientError(404, 'no grade id found. ');
+    const sql = `
+  select "gradeId","name","course","score"
```

Comment:
This is fine, but it's more conventional to just `select *` instead of naming all those columns.